can_tools/scrapers/official/NH/nh_demographics.py

```diff
@@ -0,0 +1,128 @@
+from tableauscraper import TableauScraper
+import us
+import pandas as pd
+from can_tools.scrapers import variables
+import multiprocessing
+from functools import partial
+
+from can_tools.scrapers.official.base import StateDashboard
+
+NUM_PROCESSES = 12
+
+
+class NHVaccineRace(StateDashboard):
+    has_location = False
+    source = "https://www.covid19.nh.gov/dashboard/vaccination"
+    source_name = "New Hampshire DHHS"
+    location_type = "county"
+
+    state_fips = us.states.lookup("New Hampshire").fips
+
+    variables = {
+        "Residents with At Least 1 Dose:": variables.INITIATING_VACCINATIONS_ALL,
+        "Residents Fully Vaccinated:": variables.FULLY_VACCINATED_ALL,
+    }
+
+    # hard coding to make parallel fetching easier
```

Comment:
mostly for my curiosity, why does this make it easier?  What would break if you just did `counties = self._retrieve_counties()` in the fetch() method?

Reply:
In this case there are some locations that don't exactly match the column names in the dashboard (because of how they split up Hillsborough into "rest of Hillsborough", "Nashua", and "Manchester"). Using self._retrieve_counties(), we wouldn't collect those and would be missing those locations. 

We could call `self._retrieve_counties()` and edit/append to the output to match these different locations, but I figured it was less work to just hard code them. (These hard coded values come directly from the dashboard so they match 1 to 1 with the TableauScraper selects/queries)